ParisBNF/et/BNFet616.xml

```diff
@@ -23,8 +23,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                 <msDesc xml:id="ms">
                     <msIdentifier>
                         <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
                         <collection>Griaule</collection>
-                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b53115216c">BnF Éth. 616</idno>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b53115216c">BnF BnF Éthiopien 616 (Griaule 308)</idno>
```

Comment:
1) only once BnF (now you have twice)
2)
should not Griaule be an alternative shelfmark?
see here https://archivesetmanuscrits.bnf.fr/ark:/12148/cc101792k
so we need 
Éthiopien 616 
Griaule 308 
Acquisition no. 229